prisma/schema.prisma

```diff
@@ -15,7 +15,7 @@ datasource db {
 model Customer {
   id          Int       @id @default(autoincrement())
   name        String
-  contact     Contact?
+  contact     Contact[]
```

Comment:
Curious on why you changed this relation

Reply:
After initially cloning the repo, an error caused prisma to crash. It was either this or Jules's fix of adding `@unique` the the customerId in the Contact table

Reply:
gotcha!